book/publishing/about/aims-and-scope.md

```diff
@@ -0,0 +1,30 @@
+(pb-about-aimsscope)=
+
+# Aims and Scope
+We consider submissions from all areas of environmental science. This includes (but it is not restricted to):
+* introducing relevant environmental datasets irrespective of their modality (image, labels, points, shapes, surface text).
+* describing the outputs of a machine learning/computer vision model suited to our understanding of Planet Earth.
+* showcasing an open-source software suited to Environmental Data Science.
+* pre- or post-processing routines relevant for Environmental Science. 
+
+Submissions could be at any stage from notebook idea to a working prototype from an existing GitHub repository. 
+The notebook should address scientific and/or technical aspects that EDS book audience could adopt, reuse, and/or extend for their purposes.
+
+The ideal notebook length is XX (TBD) lines, although in certain exceptions long notebooks are also acceptable.
```

Comment:
```suggestion
The ideal notebook length is between 15-20 notebook cells, although in certain exceptions long notebooks are also acceptable.
```

Reply:
I'll double check how many cells previous notebooks had. Not sure if we can retrieve automatically an estimated number of code lines instead of number of cells? hope to learn it with my attendance of the Jupyter notebook file workshop next week!

Reply:
According to a quick assessment over all published EDS book notebooks using [cloc](https://github.com/AlDanial/cloc), I'd suggest to indicate the optimal range is between 100 and 500 for physical lines of code in Code Cells, and 500 to 5000 comments in Markdown Cells.

Reply:
Fixed in https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/ecb175e228a21059ff0dcdbeee0f17bfaf763976.